lab01.py

```diff
@@ -1,4 +1,27 @@
 integer_1 = int(input("Enter one integer: "))
 integer_2 = int(input("Enter one integer: "))
 integer_3 = int(input("Enter one more integer: "))
-print("You put in:", integer_1, integer_2, integer_3)
\ No newline at end of file
+print("You put in:", integer_1, integer_2, integer_3)
+added = integer_1 + integer_2 + integer_3
+print("Sum =", added)
+product = integer_1 * integer_2 * integer_3
+print("Product =", product)
+power = added**integer_1
+print("Sum to the power of the first integer:", power)
+sum_divided_int2 = added / integer_2
+print("Sum divided by second integer: ", sum_divided_int2)
+sum_divided_int3 = added // integer_3
+print("Sum divided by integer 3:", sum_divided_int3)
```

Comment:
The output of this line is `Sum divided by integer 3: 2`

That's a bit confusing...the line is describing the sum divided by the _third integer_ but the text kind of sounds like it's the sum divided by the number 3. Would recommend using the same verbiage as line 12 ("second integer"...."third integer")